microsoft/testsuites/performance/storageperf.py

```diff
@@ -142,6 +343,8 @@ def _perf_premium_datadisks(
         partition_disks = _format_disk(node, data_disks)
         _stop_raid(node)
         _make_raid(node, partition_disks)
+        mount = node.tools[Mount]
+        mount.mount("/dev/md0", "/data", options="nobarrier")
```

Comment:
use the filename variable of below line.

Reply:
This change is not required. I forgot to remove it.